File: Python/2551.py
primeiro=0
recorde=0
while True:
    try:
        n=int(input())
        for i in range(n):
            linha=input().split(" ")
            tempo=int(linha[0])
            tempo/=60
            dist=int(linha[1])
            if (primeiro==0):
                print("1")
                primeiro=1
            if (recorde==0):
                recorde=dist/tempo
                dia=i+1                
            elif (recorde<(dist/tempo)):
                recorde=dist/tempo
                print(i+1)
            primeiro=0
            recorde=0
    except (EOFError):
        break

File: C/2551.c
#include <stdio.h>
int main() {
    int n, i, x, y, dia=1, primeiro=0;
    double melhor=0.0;

    while ((scanf("%d", &n)!=EOF)) {
        for (i=1; i<=n; i++) {
            scanf("%d %d", &x, &y);
            if (melhor==0.0) melhor=(x*1.0/y);
            if (melhor>(x*1.0/y)) {
                melhor=(x*1.0/y);
                dia=i;
            }
        }
        if (primeiro==0) {
            printf("1\n");
            primeiro=1;
        }
        printf("%d\n", dia);
        dia=1;
        melhor=0.0;
    }
    return 0;
}
